scripts/shared/lib/clusters_kind

```diff
@@ -261,11 +257,47 @@ function download_ovnk() {
     )
 }
 
+function prepare_ovn_ic() {
```

Comment:
If this is only to prepare the env' for OVN to build images from a specific commit, its better to rename it as prepare_for_ovn() or prepare_ovn()